bot.user.js

```diff
@@ -311,17 +311,19 @@ document.onkeydown = function (e) {
             window.savePreference('collisionDetection', window.collisionDetection);
         }
 
-        // Letter 'A' to increase collision detection radius
-        if (e.keyCode === 65) {
+          // Letter 'A' to increase collision detection radius
+        if (e.keyCode === 65 && window.snake.sp < 10) {
```

Comment:
Is there a technical reason to not allow adjusting collision radius while accelerated?